backend/src/Data/Database.cs

```diff
@@ -1,6 +1,7 @@
-using System;
+﻿using System;
+using Database.GraphQl.Databases;
 
-namespace Database.GraphQl.Databases;
```

Comment:
The package `Data` should only contain domain models whose data is stored in the PostgreSQL database. This model is not stored in PostgreSQL but just fetched from the Metabase. Has this changed? If not, then please put it back into `GraphQl.Databases`.